### DISCUSSION TITLE
Show what is being retrieved from Org Metadata Browser

### DISCUSSION THREAD

**Discussion by pranavraulkar** [forcedotcom/salesforcedx-vscode#5858] [Ideas]
Right now, we are able to concurrently download metadata from the org metadata browser.

However, it does not show what is being downloaded, while its being downloaded. When downloading multiple metadata concurrently, one has to wait and look at the output window to see what request has completed.

<img width="560" alt="SCR-20240305-klhw" src="https://github.com/forcedotcom/salesforcedx-vscode/assets/1922019/844ec4e6-0ccf-43bc-8cdf-d4eabd189dd2">

It'd be good to show whats being downloaded so one need not wait for results in output window.

<img width="560" alt="SCR-20240305-klhw-2" src="https://github.com/forcedotcom/salesforcedx-vscode/assets/1922019/dff5f952-babb-47ad-8825-93cc4f5cc11a">

**Comment by AnanyaJha** [comment]
Thanks for the feature request @pranavraulkar ! We'll keep this in mind for future enhancements to the Org Browser.

**Comment by mshanemc** [comment]
done!
